### PR TITLE
Add links to w3 "An alt Decision Tree tutorial" in docs

### PR DESCRIPTION
**Pull request by jolka-ef**
### Description

Add links to w3 "An alt Decision Tree tutorial" in docs

### Related issues and pull requests
Relates to #36046

### CODE REVIEW

File: files/en-us/learn/accessibility/html/index.md [hunk 1]
```diff
@@ -510,6 +510,9 @@ Any personal knowledge or extra description shouldn't be included here, as it is
 
 One thing to consider is whether your images have meaning inside your content, or whether they are purely for visual decoration, and thus have no meaning. If they are decorative, it is better to write an empty text as a value for `alt` attribute (see [Empty alt attributes](#empty_alt_attributes)) or to just include them in the page as CSS background images.
 
+> [!NOTE]
+> Read [An alt Decision Tree](https://www.w3.org/WAI/tutorials/images/decision-tree/) to learn how to use an alt attribute for images.
+
 > [!NOTE]
 > Read [Images in HTML](/en-US/docs/Learn/HTML/Multimedia_and_embedding/Images_in_HTML) and [Responsive images](/en-US/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images) for a lot more information about image implementation and best practices.
```
Comment (bsmth) [inline files/en-us/learn/accessibility/html/index.md:517]: We can combine these notes here as it usually looks and reads better than two notes side-by-side:

```suggestion
> [!NOTE]
> Read [Images in HTML](/en-US/docs/Learn/HTML/Multimedia_and_embedding/Images_in_HTML) and [Responsive images](/en-US/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images) for more information about image implementation and best practices.
> You can also check [An alt Decision Tree](https://www.w3.org/WAI/tutorials/images/decision-tree/) to learn how to use an alt attribute for images in various situations.
```

File: files/en-us/learn/getting_started_with_the_web/html_basics/index.md [hunk 2]
```diff
@@ -127,7 +127,8 @@ The keywords for alt text are "descriptive text". The alt text you write should
 Try coming up with some better alt text for your image now.
 
 > [!NOTE]
-> Find out more about accessibility in our [accessibility learning module](/en-US/docs/Learn/Accessibility).
+> Find out more about accessibility in our [accessibility learning module](/en-US/docs/Learn/Accessibility) and how
+> to use [alt attribute in different scenarios](https://www.w3.org/WAI/tutorials/images/decision-tree/)
```
Comment (bsmth) [inline files/en-us/learn/getting_started_with_the_web/html_basics/index.md:131]: Copying across the wording from above :)

```suggestion
> Find out more about accessibility in our [accessibility learning module](/en-US/docs/Learn/Accessibility) and [An alt Decision Tree](https://www.w3.org/WAI/tutorials/images/decision-tree/) to learn how to use an alt attribute for images in various situations.
```

File: files/en-us/learn/html/multimedia_and_embedding/images_in_html/index.md [hunk 3]
```diff
@@ -118,7 +118,7 @@ What exactly should you write inside your `alt` attribute? It depends on _why_ t
 Essentially, the key is to deliver a usable experience, even when the images can't be seen. This ensures all users are not missing any of the content. Try turning off images in your browser and see how things look. You'll soon realize how helpful alt text is if the image cannot be seen.
 
 > [!NOTE]
-> For more information, see our guide to [Text Alternatives](/en-US/docs/Learn/Accessibility/HTML#text_alternatives).
+> For more information, see our guide to [Text Alternatives](/en-US/docs/Learn/Accessibility/HTML#text_alternatives) and [An alt Decision Tree](https://www.w3.org/WAI/tutorials/images/decision-tree/)
```
Comment (bsmth) [inline files/en-us/learn/html/multimedia_and_embedding/images_in_html/index.md:121]:
```suggestion
> For more information, see our guide to [Text Alternatives](/en-US/docs/Learn/Accessibility/HTML#text_alternatives) and [An alt Decision Tree](https://www.w3.org/WAI/tutorials/images/decision-tree/) to learn how to use an alt attribute for images in various situations.
```

File: files/en-us/learn/tools_and_testing/cross_browser_testing/accessibility/index.md [hunk 4]
```diff
@@ -186,7 +186,8 @@ Sometimes it is not possible to avoid losing keyboard accessibility. You might h
 Text alternatives are very important for accessibility — if a person has a visual or hearing impairment that stops them being able to see or hear some content, then this is a problem. The simplest text alternative available is the humble `alt` attribute, which we should include on all images that contain relevant content. This should contain a description of the image that successfully conveys its meaning and content on the page, to be picked up by a screen reader and read out to the user.
 
 > [!NOTE]
-> For more information, read [Text alternatives](/en-US/docs/Learn/Accessibility/HTML#text_alternatives).
+> For more information, read [Text alternatives](/en-US/docs/Learn/Accessibility/HTML#text_alternatives) and
+> [An alt Decision Tree](https://www.w3.org/WAI/tutorials/images/decision-tree/)
```
Comment (bsmth) [inline files/en-us/learn/tools_and_testing/cross_browser_testing/accessibility/index.md:190]:
```suggestion
> For more information, read [Text alternatives](/en-US/docs/Learn/Accessibility/HTML#text_alternatives) and [An alt Decision Tree](https://www.w3.org/WAI/tutorials/images/decision-tree/) to learn how to use an alt attribute for images in various situations.
```

File: files/en-us/mdn/writing_guidelines/howto/images_media/index.md [hunk 5]
```diff
@@ -92,6 +92,8 @@ Examples:
 
 While purely decorative images should have an empty `alt`, images added to MDN documentation should have a purpose, and therefore require a non-empty-string description.
 
+- [An alt Decision Tree • Images • WAI Web Accessibility Tutorials](https://www.w3.org/WAI/tutorials/images/decision-tree/)
+
```
Comment (bsmth) [inline files/en-us/mdn/writing_guidelines/howto/images_media/index.md:96]:
```suggestion
While purely decorative images should have an empty `alt`, images added to MDN documentation should have a purpose, and therefore require a non-empty-string description.
For hints on alt text, see [An alt Decision Tree](https://www.w3.org/WAI/tutorials/images/decision-tree/) to learn how to use an alt attribute for images in various situations. 

```

**Comment by bsmth** [comment]
Thank you! A couple of wording suggestions, but overall looking great 👍🏻

File: files/en-us/web/api/htmlinputelement/alt/index.md [hunk 6]
```diff
@@ -40,3 +40,4 @@ inputElement.alt = "A much better description";
 - {{HTMLElement("input")}}
 - {{HTMLElement("img")}}
 - [Writing good alt text](https://www.wcag.com/blog/good-alt-text-bad-alt-text-making-your-content-perceivable/)
+- [An alt Decision Tree](https://www.w3.org/WAI/tutorials/images/decision-tree/)
```
Comment (dipikabh) [inline files/en-us/web/api/htmlinputelement/alt/index.md:43]: We can add a bit more info to make these consistent with our "See also" guidelines:
```suggestion
- [Good alt text, bad alt text — Making your content perceivable](https://www.wcag.com/blog/good-alt-text-bad-alt-text-making-your-content-perceivable/) on WCAG.com (2021)
- [An alt Decision Tree](https://www.w3.org/WAI/tutorials/images/decision-tree/) on W3C Web Accessibility Initiative (WAI)
```

**Comment by bsmth** [comment]
Hi @jolka-ef are you coming back to this one? There's a few suggestions open for you to take a look at. Otherwise, are you okay with us making the changes on your branch and merging? Thanks!

**Comment by jolka-ef** [comment]
Hi @bsmth !
Sorry for not getting back to this sooner. I will do it today :)
Thanks !

**Comment by bsmth** [comment]
> Hi @bsmth ! Sorry for not getting back to this sooner. I will do it today :) Thanks !

No problem, just checking in!

**Review by bsmth — APPROVED** [review]
Super, thank you! 👍🏻

**Comment by jolka-ef** [comment]
My pleasure :smiley:

**Pull request merged**